warp/_src/builtins.py

```
@@ -5638,6 +5638,22 @@ def bvh_query_next_tiled_dispatch_func(input_types: Mapping[str, type], return_t
     is_differentiable=False,
 )
 
+add_builtin(
+    "mesh_get_group_root",
+    input_types={"id": uint64, "group": int},
```

Ruff (0.14.7)

`uint64` may be undefined, or defined from star imports

(F405)